addons/packages/contour/1.21.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: docker.io/envoyproxy/envoy:v1.22.0
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v1.22.0
+          url: docker.io/envoyproxy/envoy:v1.22.0
+      - preresolved:
+          url: index.docker.io/envoyproxy/envoy@sha256:e9ad4cde73ffb01402c4e3fb6ce1a764d9e84650230135b88495f30904a378fc
+  image: index.docker.io/envoyproxy/envoy@sha256:e9ad4cde73ffb01402c4e3fb6ce1a764d9e84650230135b88495f30904a378fc
```

Comment:
This doesn't look to have changed from the 1.20.1 package, the `make lock-package-images` should update this but if it's not, need to dig into what's going wrong

Reply:
hm, yeah that command only added that `preresolved` section and I didn't really think about it, I'll look into it

Reply:
Yeah I feel like maybe i've run into this before and probably failed to document - something with docker auth I think?

Reply:
hm, i should be logged in and nothing i seem to do, short of change the image sha is changing anything, `make lock-package-images` keeps just adding a new `preresolved` list element

Reply:
updated manually for now, will ask if theres something we need to do here

Reply:
ah, i think i figured it out, I had done a find-replace on the whole 1.21.0 package and that included the imgpkg lock file, when I reverted it, the images were resolved correctly

maybe something to bring up in kbld, not to be so trusting even if you find the lock file has a matched image?